### PR TITLE
[#86] Add roundtrip property test for Repo

### PR DESCRIPTION
**Pull request by rashadg1030**

### CODE REVIEW

File: test/Spec.hs [hunk 1]
```diff
@@ -24,6 +25,7 @@ hspecTests = sequential . joinSpecs "issue-wanted" []
 hedgehogTests :: AppEnv -> Group
 hedgehogTests env = Group "Roundtrip properties" 
     [ issueRoundtripProp env `named` "fromRow . toRow ≡ id"
+    , repoRoundtripProp env `named` "fromRow . toRow ≡ id"
```
Comment (chshersh) [inline test/Spec.hs]: In terminal you will this text. So let's add some context to it to understand which test this is for:

```haskell
"Issue: fromRow . toRow ≡ id"
"Repo: fromRow . toRow ≡ id"
```

File: test/Test/Core/Repo.hs [hunk 2]
```diff
@@ -1 +1,65 @@
...
+    pure Repo{..} 
+  where
+    genId :: MonadGen m => m (Id Repo)
+    genId = Id <$> Gen.int (Range.constant 1 500)
+
+    genOwner :: MonadGen m => m RepoOwner
+    genOwner = RepoOwner <$> Gen.text (Range.constant 1 20) Gen.alphaNum
+    
+    genName :: MonadGen m => m RepoName
+    genName = RepoName <$> Gen.text (Range.constant 1 20) Gen.alphaNum
```
Comment (chshersh) [inline test/Test/Core/Repo.hs]: Let's create `Test.Gen` module and move all common generators for `Repo` and `Issue` there, so we can just import them and reuse.

Comment (rashadg1030) [reply test/Test/Core/Repo.hs]: Is there some way I can factor out `genId` too?

Comment (chshersh) [reply test/Test/Core/Repo.hs]: You can just generate `Id a`. For tests it doesn't matter, what type parameter is there

**Comment by rashadg1030** [comment]
Made suggested changes

**Pull request merged**